query-model/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/querymodel/jpql/expression/impl/LocalDateTimeExpression.kt

```diff
@@ -0,0 +1,6 @@
+package com.linecorp.kotlinjdsl.querymodel.jpql.expression.impl
+
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expression
+import java.time.LocalDateTime
+
+object LocalDateTimeExpression : Expression<LocalDateTime>
```

Comment:
Please add the `Internal annotation` because this class belongs to an implementation.

```kotlin
@Internal
object LocalDateTimeExpression : Expression<LocalDateTime>
```